src/com/amazon/ion/util/Equivalence.java

```diff
@@ -61,6 +61,20 @@
  *</pre>
  *
  * <p>
+ * Additional options are available by configuring an <code>Equivalence</code>
```

Comment:
To streamline things, why not `{@link}` to `Equivalence` and then use the simple class name in the example rather than the fully qualified class name?

Reply:
> why not {@link} to Equivalence

Because the link would point back to the class documentation (i.e. this).

> and then use the simple class name in the example rather than the fully qualified class name?

Just for consistency with the existing style. See the example above this one.

Reply:
> Because the link would point back to the class documentation (i.e. this).

Oops, didn't see which file this was in. My mistake!